eras/allegra/impl/cardano-ledger-allegra.cabal

```
@@ -67,7 +67,7 @@ library
         bytestring,
         cardano-crypto-class,
         cardano-ledger-binary ^>=1.3,
-        cardano-ledger-core >=1.13 && <1.15,
+        cardano-ledger-core >=1.13 && <1.16,
```

It is the lower bound that needs to be updated. `cardano-ledger-core-1.14.0.0` adds a function that `cardano-ledger-allegra` depends on. Which means it will no longer work with `cardano-ledger-core-1.13.x`
```suggestion
        cardano-ledger-core >=1.14 && <1.15,
```